db/sql/3_msar_custom_aggregates.sql

```diff
@@ -132,3 +133,143 @@ Link: https://github.com/centerofci/mathesar/pull/2981
   finalfunc = msar.point_to_time,
   initcond = '(0,0)'
 );
+
+
+CREATE OR REPLACE FUNCTION 
+msar.month_to_degrees(date_ DATE) returns DOUBLE PRECISION AS $$/*
+Convert discrete month to degrees.
+
+To get the fraction of 12 months passed, we extract the month from date, subtract 1 from 
+the result to confine it in range [0, 12) and divide by 12, then to get the equivalent
+fraction of 360°, we multiply by 360, which is equivalent to multiply by 30.
+
+Examples:
+  2023-05-02 => 120
+  2023-07-12 => 180
+  2023-01-01 =>   0
+  2023-12-12 => 330
+*/
+SELECT ((EXTRACT(MONTH FROM date_) - 1)::double precision) * 30;    
+$$ LANGUAGE SQL;
+
+
+CREATE OR REPLACE FUNCTION 
+msar.degrees_to_month(degrees DOUBLE PRECISION) RETURNS INT AS $$/*
+Convert degrees to discrete month.
+
+To get the fraction of 360°, we divide degrees value by 360 and then to get the equivalent 
+fractions of 12 months, we multiply by 12, which is equivalent to divide by 30.
+
+Examples:
+     0 =>  1
+   120 =>  5
+   240 =>  9
+   330 => 12
+  -120 =>  9
+*/
+SELECT (((degrees::numeric % 360 + 360) % 360)::double precision / 30)::int + 1;
+$$ LANGUAGE SQL;
+
+
+CREATE OR REPLACE FUNCTION 
+msar.month_int_to_string(month_ INT) RETURNS TEXT AS $$/*
+Convert integer representing month to string
+
+Examples:
+   0 => January
+   1 => February
+   and so on...
+*/
+SELECT CASE
+  WHEN month_ = 1 THEN 'January'
+  WHEN month_ = 2 THEN 'February'
+  WHEN month_ = 3 THEN 'March'
+  WHEN month_ = 4 THEN 'April'
+  WHEN month_ = 5 THEN 'May'
+  WHEN month_ = 6 THEN 'June'
+  WHEN month_ = 7 THEN 'July'
+  WHEN month_ = 8 THEN 'August'
+  WHEN month_ = 9 THEN 'September'
+  WHEN month_ = 10 THEN 'October'
+  WHEN month_ = 11 THEN 'November'
+  WHEN month_ = 12 THEN 'December'
+END;
+$$ LANGUAGE SQL;
```

Comment:
If you'll recall, we decided to keep the months as integers, since these won't be localized.